src/libraries/Microsoft.CSharp/ref/Microsoft.CSharp.csproj

```diff
@@ -1,6 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);nullable</NoWarn>
```

Comment:
Do nullable warnings need to be suppressed in the ref?  I'd have expected this to only be needed in the src.

Reply:
There are no warnings to suppress, so I removed this property.